src/components/FileItem.tsx

```diff
@@ -174,7 +182,7 @@ export class FileItem extends React.PureComponent<IFileItemProps> {
         }
         onDoubleClick={this.props.onDoubleClick}
         style={this.props.style}
-        title={`${this.props.file.to} ● ${status}`}
+        title={`${this.props.file.to} ● ${this.props.trans.__(status)}`}
```

Comment:
We might need to put the whole string inside trans. Maybe on some language it makes sense to put things backwards?

Something like this (I may have written it incorrectly, but you get the idea I hope :)

```
{`${this.props.trans.__(' %1 ● status', this.props.file.to)}`}
```